base_exception/views/base_exception_view.xml

```diff
@@ -0,0 +1,111 @@
+            </tree>
+        </field>
+    </record>
+    <record id="view_exception_rule_form" model="ir.ui.view">
+        <field name="name">exception.rule.form</field>
+        <field name="model">exception.rule</field>
+        <field name="arch" type="xml">
+            <form string="Exception Rule Setup" name="exception_rule">
+                <sheet>
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        invisible="active"
+                    />
+                    <div class="oe_button_box" name="button_box">
+                    </div>
+                    <div class="oe_title">
+                        <label for="name" class="oe_edit_only" />
+                        <h1>
+                            <field name="name" />
+                        </h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="active" invisible="1" />
+                            <field name="description" />
+                        </group>
+                    </group>
+                    <field name="exception_type" invisible="1" />
+                    <group>
+                        <group colspan="4" groups="base.group_system">
+                            <field name="model" />
+                            <field name="exception_type" widget="radio" />
+                            <field
+                                name="domain"
+                                invisible="exception_type != 'by_domain'"
+                                required="exception_type == 'by_domain'"
+                                widget="domain"
+                                options="{'model': 'model'}"
+                            />
+                            <field name="is_blocking" />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page
+                            name="code"
+                            string="Python Code"
+                            invisible="exception_type != 'by_py_code'"
+                            required="exception_type == 'by_py_code'"
+                        >
+                            <field
+                                name="code"
+                                widget="ace"
+                                options="{'mode': 'python'}"
+                                placeholder="Enter Python code here. Help about Python expression is available in the help tab of this document."
+                            />
+                        </page>
+                        <page
+                            name="help"
+                            string="Help"
+                            invisible="exception_type != 'by_py_code'"
+                        >
+                            <group>
```

Comment:
the view is broken, we should remove the `<group> `
![image](https://github.com/OCA/server-tools/assets/8724793/95958ba7-73e5-49f8-9d1b-094ef2e85013)

Reply:
It's ready